source/Interfaces/NetWorth/NetworkInterface.hpp

```diff
@@ -0,0 +1,93 @@
+/**
+ * This file is part of the Fall 2023, CSE 491 course project.
+ * @brief A networking interface that allows information to be sent across a  network
+ * @note Status: PROTOTYPE
+ **/
+
+#pragma once
+
+#include <cassert>
+#include <string>
+#include <vector>
+#include <SFML/Network/UdpSocket.hpp>
+#include <SFML/Network/Packet.hpp>
+
+#include "../../core/InterfaceBase.hpp"
+
+using namespace sf;
+
+namespace cse491 {
+    namespace netWorth{
+    /**
+     * TODO: Delete or incorporate
+     */
+        class NetworkingInterface{
+        private:
+
+        protected:
+            UdpSocket mSocket; ///The socket we are going to make our connection
```

Comment:
With the new format we just discussed in class, I think member variables can still begin with m, but we should probably change these to something like `m_socket` for consistency.